lib/client/config.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Client
+  # The Config class holds configuration settings for the LocalizeRubyClient.
+  # It provides attributes for app_id, private_key, locales_dir_path, site, and api_version.
+  # It also includes a method to validate these configurations to ensure they are properly set.
+  class Config
+    attr_accessor :app_id, :project_uid, :private_key, :locales_dir_path, :site, :api_version
+
+    def initialize
+      @app_id = nil
+      @private_key = nil
+      @project_uid = nil
+      @locales_dir_path = Rails.root.join("config", "locales").to_s
+      @site = "http://localhost:6002/api"
```

Comment:
I would suggest to put the production site API by default 

```suggestion
      @site = "https://localize.cirro.io/api"
```

Reply:
Yes 😅